action.yml

```diff
@@ -46,30 +65,43 @@ runs:
       with:
         codeball-job-id: ${{ steps.codeball_baller.outputs.codeball-job-id }}
 
-    # If Codeball approved the contribution, add a "codeball:approved" label
+    # If the Codeball confidence is high, add the "codeball:approved" label
     - name: Label Approved
       uses: sturdy-dev/codeball-action/labeler@v2
-      if: ${{ steps.codeball_status.outputs.approved == 'true' && inputs.labelPullRequestsWhenApproved == 'true' && steps.codeball_status.outputs.jobType == 'contribution' }}
+      if: ${{ steps.codeball_status.outputs.jobType == 'contribution' && inputs.labelPullRequestsWhenApproved == 'true' && steps.codeball_status.outputs.confidence >= inputs.approveThreshold  }}
       with:
         name: "codeball:approved"
         color: "86efac" # green
-        remove-label-names: "codeball:needs-review"
+        remove-label-names: "codeball:needs-review,codeball:needs-careful-review"
         codeball-job-id: ${{ steps.codeball_baller.outputs.codeball-job-id }}
 
-    # If Codeball did not approve the contribution, add a "codeball:needs-review" label
+
+    # If the Codeball confidence is medium, add the "codeball:needs-review" label
     - name: Label Needs Review
       uses: sturdy-dev/codeball-action/labeler@v2
-      if: ${{ steps.codeball_status.outputs.approved == 'false' && inputs.labelPullRequestsWhenReviewNeeded == 'true' && steps.codeball_status.outputs.jobType == 'contribution' }}
+      if: ${{ steps.codeball_status.outputs.jobType == 'contribution' && inputs.labelPullRequestsWhenReviewNeeded == 'true' && steps.codeball_status.outputs.confidence >= inputs.carefulReviewThreshold && steps.codeball_status.outputs.confidence < inputs.approveThreshold  }}
```

Comment:
shouldn't it be
```suggestion
      if: ${{ steps.codeball_status.outputs.jobType == 'contribution' && inputs.labelPullRequestsWhenReviewNeeded == 'true' && steps.codeball_status.outputs.confidence <= inputs.carefulReviewThreshold && steps.codeball_status.outputs.confidence < inputs.approveThreshold  }}
```

Reply:
I don't think so. There is now three different labels: `codeball:needs-careful-review`, `codeball:needs-review`, and `codeball:approved`. This step is for needs-review, which happens when the score is between the two configured thresholds.